lib/routes/index.js

```diff
@@ -59,6 +60,7 @@ module.exports = function (
   )
   const session = require('./session')(log, isA, error, db)
   const sign = require('./sign')(log, P, isA, error, signer, db, config.domain, devices)
+  const smsRoute = require('./sms')(log, isA, error, config, customs, smsImpl)
```

Comment:
Hrm, `sms` is the only name that comes to mind lol. It is fine not to change, just stood out among the other names.